### ISSUE TITLE
DP #104 / Storyboard - Decentralized news, social media, trading site. 

### ISSUE THREAD

**Issue by Splitspace** [district0x/district-proposals#106]
## Name
**Storyboard**. To me, the name "Storyboard" describes the general feel and appearance of the District. If you guys have any suggestions, feel free to bring them up!

## Purpose
A decentralized social media site, Reddit-esque, where users submit content which can then be voted on, rated and optionally purchased by users, and a dividend is paid to the content creators. 

## Description
The site, similar to many other sites, is a place where users can come together and form communities. The site would be categorized into moderated categories based on content type. What separates this from other social media sites, is that a post on Storyboard can simultaneously be a piece of content, a market listing or both. 
Example submissions: user could submit a piece of artwork, and users would not only be able to see the artwork, but purchase a high resolution version of it.

**Comment by mckmuze** [comment]
Interesting. So are you wanting to take a curation market approach with this? Nice mock ups BTW!

**Comment by Splitspace** [comment]
@mckmuze Yep, discussed this in the slack, but just updated the proposal with a detailed explanation. More mockups coming soon too!

**Comment by Alexand0x** [comment]
Love the basic premise of a curated content market - "a place where content creators can share their content and profit from it, even without directly selling it" - this is a really powerful idea.

Mockups are great, keep them coming!

**Comment by mckmuze** [comment]
@Splitspace Can you get back to me with a token address please.

**Comment by Bradymck** [comment]
Hey @Splitspace 

I'm not sure if you're aware but we recently launched a new bounty to migrate your proposals to the actual District Registry: https://registry.district0x.io/

We're replacing the old voting app with the registry. Let me know if you need help but I would love to see you migrate this over so you can claim your DNT.

It does take a 10,000 DNT submit to submit your proposal but this gives you an extra 2000 you can stake in the registry beyond the deposit amount.

Hit me up on Telegram or Discord if you need help or have questions.

Telegram: https://t.me/district0x/75217
Discord: https://discord.gg/P9RQejv

PS, please excuse the canned response. I am encouraging everyone here to start migrating so they can claim their 12000 DNT.
